core/src/main/kotlin/entity/channel/Channel.kt

```diff
@@ -26,6 +27,9 @@ public interface Channel : ChannelBehavior {
      */
     public val type: ChannelType get() = data.type
 
+    /** The flags of this channel, if present. */
+    public val flags: ChannelFlags? get() = data.flags.value
```

Comment:
So far, only forum threads have flags, wouldn't it be better to move it to his entity?

Reply:
I think for flags it's ok to be located in the base `Channel` type. They might add new `ChannelFlag`s in the future...